android/src/org/mavlink/qgroundcontrol/QGCActivity.java

```diff
@@ -76,7 +77,8 @@
     private static PendingIntent                        _usbPermissionIntent = null;
     private TaiSync                                     taiSync = null;
     private Timer                                       probeAccessoriesTimer = null;
-
+    private static WifiManager.MulticastLock            _cLock;
```

Comment:
Could you change this to something a bit more descriptive like '_wifiMulticastLog' or something. If you just see '_cLock' further down the code its not clear at a glance what it is.